cli/types/cypress.d.ts

```diff
@@ -1163,6 +1163,15 @@ declare namespace Cypress {
      */
     end(): Chainable<null>
 
+    ensureElement(subject: any, commandName: string): void
```

Comment:
`any => void` is correct here, even for `ensureAttached`, which really only makes sense with DOM elements: you can pass in any subject, and these functions either do nothing and return nothing, or they throw an error.